content/blogs/newsletter-100.md

```diff
@@ -0,0 +1,64 @@
+---
+title:       "Newsletter 100 - January 2021 news"
+author:      "Extravert34"
+date:        2021-02-06
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers!
+This is an attempt to restore the tradition of annual newsletters covering our resent development efforts.
```

Comment:
```suggestion
This is an attempt to restore the tradition of annual newsletters covering our recent development efforts.
```

Reply:
Also "annual" = "yearly". You probably mean "recurring".